src/sugar3/graphics/radiotoolbutton.py

```diff
@@ -29,7 +35,27 @@
 
 
 class RadioToolButton(Gtk.RadioToolButton):
-    """An implementation of a "push" button."""
+    '''
+    An implementation of a "push" button. Properies that can
```

Comment:
Also, I don't think that the description really explains that these are radio buttons (not just ToolButtons)